data/scenarios/Testing/699-teleport-blocked.yaml

```diff
@@ -0,0 +1,45 @@
+name: Teleport to blocked
+description: |
+  Teleporting self or another robot to blocked location results in destruction of that robot.
+  https://github.com/swarm-game/swarm/issues/475
+creative: true
+objectives:
+  - condition: |
+      try {
+        try {
+          robotNamed "one"
+        } {
+          robotNamed "two"
+        };
+        return false;
+      } {
+        return true; // both robots must be destroyed
+      }
+solution: |
+  o <- robotNamed "one"; reprogram o {t <- robotNamed "two"; teleport t (3,0); teleport self (3,0)}
```

Comment:
@byorgey what do you think about this test? 🙂

Does it cover the original issue enough? I guess I could test `move` and surviving base too. 🤔

Reply:
I like it!  If we want to test additional stuff maybe we should put that in a separate test.

Reply:
Done :+1: